worker/internal/benthos/transformers/transform_email.go

```diff
@@ -44,9 +45,29 @@ func init() {
 			return nil, err
 		}
 
+		eL, err := args.Get("exclusion_list")
+		if err != nil {
+			return nil, err
+		}
+
+		excl, ok := eL.([]any)
+		if !ok {
+			return nil, fmt.Errorf("error")
```

Comment:
just so this doesn't get lost: this error should be updated. 
Should do something like this:

```
fmt.Errorf("Expected []any, got: %T", excl)
```

Reply:
updated